mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/installer_tagdata.cpp

```diff
@@ -53,8 +53,10 @@ bool InstallerTagData::Init(const std::wstring& cmd) {
 
 // static
 void InstallerTagData::Reset() {
-  delete current_installer_tagdata_;
-  current_installer_tagdata_ = NULL;
+  if (current_installer_tagdata_ != NULL) {
+    delete current_installer_tagdata_;
+    current_installer_tagdata_ = NULL;
+  }
```

Comment:
Too much for rewrite here. Also will add needs in C++11 standard, while we have nothing from it in this project. I think we can process without it for now, but I will count this moment on next refactoring or bugfixing.